### ISSUE TITLE
Can't export DNS records from account

### ISSUE THREAD

**Issue by garrettgalow** [cloudflare/cf-terraforming#11]
latest version of cf-terraforming

`go version`
`go version go1.11.5 darwin/amd64`

`go run cmd/cf-terraforming/main.go --email $CLOUDFLARE_EMAIL --key $CLOUDFLARE_KEY -a cf158db8694a70604a9916976a58e22b record`
```
# github.com/cloudflare/cf-terraforming/internal/app/cf-terraforming/cmd
internal/app/cf-terraforming/cmd/zone.go:53:23: impossible type assertion:
	*schema.Provider does not implement "github.com/terraform-providers/terraform-provider-cloudflare/vendor/github.com/hashicorp/terraform/terraform".ResourceProvider (wrong type for Apply method)
		have Apply(*"github.com/hashicorp/terraform/terraform".InstanceInfo, *"github.com/hashicorp/terraform/terraform".InstanceState, *"github.com/hashicorp/terraform/terraform".InstanceDiff) (*"github.com/hashicorp/terraform/terraform".InstanceState, error)
		want Apply(*"github.com/terraform-providers/terraform-provider-cloudflare/vendor/github.com/hashicorp/terraform/terraform".InstanceInfo, *"github.com/terraform-providers/terraform-provider-cloudflare/vendor/github.com/hashicorp/terraform/terraform".InstanceState, *"github.com/terraform-providers/terraform-provider-cloudflare/vendor/github.com/hashicorp/terraform/terraform".InstanceDiff) (*"github.com/terraform-providers/terraform-provider-cloudflare/vendor/github.com/hashicorp/terraform/terraform".InstanceState, error)
internal/app/cf-terraforming/cmd/zone.go:54:34: impossible type assertion:
	*schema.Provider does not implement "github.com/terraform-providers/terraform-provider-cloudflare/vendor/github.com/hashicorp/terraform/terraform".ResourceProvider (wrong type for Apply method)
		have Apply(*"github.com/hashicorp/terraform/terraform".InstanceInfo, *"github.com/hashicorp/terraform/terraform".InstanceState, *"github.com/hashicorp/terraform/terraform".InstanceDiff) (*"github.com/hashicorp/terraform/terraform".InstanceState, error)
		want Apply(*"github.com/terraform-providers/terraform-provider-cloudflare/vendor/github.com/hashicorp/terraform/terraform".InstanceInfo, *"github.com/terraform-providers/terraform-provider-cloudflare/vendor/github.com/hashicorp/terraform/terraform".InstanceState, *"github.com/terraform-providers/terraform-provider-cloudflare/vendor/github.com/hashicorp/terraform/terraform".InstanceDiff) (*"github.com/terraform-providers/terraform-provider-cloudflare/vendor/github.com/hashicorp/terraform/terraform".InstanceState, error)
```

**Comment by zackproser** [comment]
@garrettgalow This should be fixed in 2d66977e74655a504ac07c01274d6ad611d807e8. Please give it a shot and let me know.

**Comment by garrettgalow** [comment]
fixed in 2d66977
